Makefile

```diff
@@ -63,6 +63,14 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
+NOTICE: vendor/
+	rm -f NOTICE # `govendor license` appends...
+	govendor license -o NOTICE +v
+
+notice-up-to-date: NOTICE
```

Comment:
Maybe we could even used a more generic name like test-generate-diff and include generation of NOTICE in make generate or in go generate ?